subworkflows/local/standardisation_profiles.nf

```diff
@@ -158,11 +165,17 @@ workflow STANDARDISATION_PROFILES {
     // Have to sort by size to ensure first file actually has hits otherwise
     // the script fails
     ch_profiles_for_kraken2 = ch_input_profiles.kraken2
-                                .map { [it[0]['db_name'], it[1]] }
-                                .groupTuple(sort: {-it.size()} )
                                 .map {
-                                    [[id:it[0]], it[1]]
+                                    meta, profiles ->
+                                        def new_meta = [:]
+                                        new_meta.tool = meta.tool == 'kraken2-bracken' ? 'kraken2' : meta.tool // replace to get the right output-format description
+                                        new_meta.id = meta.tool // append so to disambiguate when we have same databases for kraken2 step of bracken, with normal bracken
+                                        new_meta.db_name = meta.tool == 'kraken2-bracken' ? "${meta.db_name}-bracken" : "${meta.db_name}" // append so to disambiguate when we have same databases for kraken2 step of bracken, with normal bracken
+                                    [ new_meta, profiles ]
                                 }
+                                .dump(tag: 'b4group')
```

Comment:
Are we keeping dumps?

```suggestion
```

Reply:
As you can see from the quality of the tagging..... nope, not the plan :laughing:  :laughing: